scene/main/window.cpp

```diff
@@ -930,6 +947,20 @@ bool Window::is_transient() const {
 	return transient;
 }
 
+void Window::set_transient_to_focused(bool p_transient_to_focused) {
+	ERR_MAIN_THREAD_GUARD;
+	if (transient_to_focused == p_transient_to_focused) {
```

Comment:
Early return unnecessary unless there's something triggered by it

Reply:
I think it keeps it in style of other methods, so it's fine. But yes, not strictly necessary.

Reply:
I would remove it since it's clearly not doing anything.